src/Contracts/Utilities/LogStyler.php

```diff
@@ -31,4 +31,13 @@ public function icon($level, $default = null);
      * @return string
      */
     public function color($level, $default = null);
+
+    /**
+     * Get strings to highlight.
+     *
+     * @param  array  $default
+     *
+     * @return string
+     */
+    public function toHighlight($default = []);
```

Comment:
Can you add the `array` typehint to the `$default` argument  ?

`public function toHighlight(array $default = []);`